lib/minitest/parallel_fork.rb

```diff
@@ -76,9 +76,11 @@ def self.__run(reporter, options)
         end
         raise
       end
-      stat_reporter.count += count
-      stat_reporter.assertions += assertions
-      stat_reporter.results.concat(results)
+      stat_reporters.each do |rep|
```

Comment:
In here, instead of using `stat_reporters.each do |rep|`:

```ruby
reporter.reporters.each do |rep|
  next unless %w'count assertions results count= assertions='.all?{|meth| rep.respond_to?(meth)}
```